### ISSUE TITLE
Introduce big O in README

### ISSUE THREAD

**Issue by jeena** [wuiscmc/algorithms_experiments#1]
You don't mention the big O notation in your README which is basically what you're describing there https://en.wikipedia.org/wiki/Big_O_notation

**Comment by wuiscmc** [comment]
Actually that was never the intention, just wanted to compare their running time and hypothesise the mathematical model. I knew from before that I was comparing something O(n^2) and O(n log(n)) (so I knew which was the winner before the race started :smile: )

**Comment by jeena** [comment]
Yes yes, I just was thinking that stating that in the README would benefit the reader :)

**Comment by wuiscmc** [comment]
Good idea :) I will also add the whole mathematical demonstration
